release-calendar/src/entities/release.ts

```diff
@@ -14,17 +14,42 @@
  * limitations under the License.
  */
 
-import {Entity, PrimaryColumn} from 'typeorm';
+import {Entity, PrimaryColumn, Column} from 'typeorm';
+
+export enum Channel {
+  LTS = 'lts',
+  STABLE = 'stable',
+  BETAONE = 'betaOne',
+  EXPERIMENTALONE = 'experimentalOne',
+  BETAOPTIN = 'betaOptin',
+  EXPERIMENTALOPTIN = 'experimentalOptin',
+  NIGHTLY = 'nightly',
+  ERROR = 'error',
```

Comment:
Perhaps we don't need this.

Reply:
Are you referring to 'error' or the nightly channel?

Reply:
error! (the line right above the comment)